crates/loro-internal/src/state/tree_state.rs

```diff
@@ -654,13 +653,24 @@ impl NodePosition {
     }
 }
 
+pub(crate) enum FiIfNotConfigured {
+    UseJitterZero,
```

Comment:
What's the different of `UseJitterZero` and `Zero`

Reply:
`Zero` means always use the value `0` as FI